src/app/docs/rich-text-display/rich-text-display-docs.component.html

```diff
@@ -0,0 +1,45 @@
+<sky-page>
+  <sky-docs-demo-page
+    moduleName="SkyRichTextDisplayModule"
+    moduleSourceCodePath="src/app/public/modules/rich-text-display/"
+    pageTitle="Rich text display"
+  >
+
+    <sky-docs-demo-page-summary>
+      <sky-alert alertType="warning">
+        This component is in beta. We do not plan to make any breaking changes to the component before the official release, but such changes are possible.
+      </sky-alert>
+      <p>
+        The rich text display component provides a safe way to display html strings with formatting on a web page.
```

Comment:
```suggestion
        The rich text display safely displays HTML strings with formatting.
```
My suggestion tightens up and capitalizes 'HTML," but I don't really understand the purpose of this component. I assume safe/safely is the key point, but looking at these docs and the demo, I don't get it. Why do we need this component? Can't developers just display HTML without it? What is this component doing to make that HTML safe and when/why would consumers want/need to use it. And is "with formatting" a particularly important point? If so, what is this component doing to be especially useful for HTML with formatting?

Reply:
Fair question. Maybe we could spell this out a bit more by unpacking the word "safe". The main reason for using this component is to sanitize the string. This component will search the string for any malicious code like script tags, and remove them. Is there a better way to say that?

Reply:
```suggestion
        The rich text display sanitizes HTML strings before displaying them. It searches HTML for any malicious code, such as `script` tags, and removes it as necessary. This allows you to safely display HTML strings from various sources.
```
How does that work? The first two sentences are rewritten based on your response, and the last sentence is just my guess as to why this is necessary.

Reply:
That works! Thank you.